baseapp/baseapp.go

```diff
@@ -300,6 +308,72 @@ func (app *BaseApp) QueryRouter() sdk.QueryRouter { return app.queryRouter }
 // GRPCQueryRouter returns the GRPCQueryRouter of a BaseApp.
 func (app *BaseApp) GRPCQueryRouter() *GRPCQueryRouter { return app.grpcQueryRouter }
 
+// RegisterGRPCServer registers gRPC services directly with the gRPC server.
+func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
```

Comment:
this function needs a test

Reply:
It is tested in integration tests.

Reply:
yeah, I was referring to unit tests mostly. Non-blocking tho

Reply:
I think it's not trivial to set up unit tests for this function. We would need to set up a mock gogogrpc.Server.

TBH I believe sticking with the integration tests for this one is easier to reason about.